tests/e2e/tests/build/styles/extract-css.ts

```diff
@@ -0,0 +1,31 @@
+import * as glob from 'glob';
+import { writeMultipleFiles, expectFileToMatch } from '../../../utils/fs';
+import { ng } from '../../../utils/process';
+import { stripIndents } from 'common-tags';
+
+export default function () {
+    return writeMultipleFiles({
+    'src/styles.css': stripIndents`
+      div { background: url("./assets/more.svg"); }
+    `,
+    'src/assets/more.svg': stripIndents`
+      <svg width="100" height="100">
+        <circle cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow" />
+      </svg>
+    `})
+    .then(() => ng('build'))
+    .then(() => expectFileToMatch('dist/styles.bundle.css',
+      /div\s*{\s*background:\s*url\(more.svg\);\s*}/))
+    .then(() => ng('build', '--prod'))
+    .then(() => glob.sync('dist/styles.*.bundle.css').find(file => !!file))
+    .then((stylesBundle) => expectFileToMatch(stylesBundle,
+      /div\s*{\s*background:\s*url\(more\..*svg\)\s*}/))
+    // build with deloyUrl
+    .then(() => ng('build', '--deploy-url=client/'))
+    .then(() => expectFileToMatch('dist/styles.bundle.css',
+      /div\s*{\s*background:\s*url\(more.svg\);\s*}/))
```

Comment:
Can you confirm the path here? If I get the issue correctly, this test would pass even without your fixe since it's just checks that there is an url that has `more.svg`.